Bender.yml

```diff
@@ -11,20 +15,21 @@ package:
     - "Davide Rossi <davide.rossi@unibo.it>"
 
 dependencies:
-  axi_mem_if : { git: git@github.com:pulp-platform/axi_mem_if.git, version: 0.2.1 }
-  axi : { git: "git@github.com:pulp-platform/axi.git", version: 0.39.1 }
-  common_verification : { git: "https://github.com/pulp-platform/common_verification.git", version: 0.2.3 }
-  register_interface : { git: "https://github.com/pulp-platform/register_interface.git", version: 0.3.8 }
-  common_cells: { git: "https://github.com/pulp-platform/common_cells.git", version: 1.32.0 }
-  axi_stream: { git: "git@github.com:pulp-platform/axi_stream.git", rev: "472751f550e3918215603e21734fe0ece3c66f79" }
+  axi : { git: "git@github.com:pulp-platform/axi.git",                                     version: 0.39.1 }
+  axi_mem_if : { git: git@github.com:pulp-platform/axi_mem_if.git,                         version: 0.2.1  }
```

Comment:
If I am not mistaken, `axi_mem_if` is not required anymore; if so, please remove

Reply:
If a "axi to mem" module is required, we have several in the `axi` repo (`axi_to_mem_*`), but iirc this was before adding the dma, when ethernet had buffers inside